funnel/models/__init__.py

```diff
@@ -1,7 +1,8 @@
 # -*- coding: utf-8 -*-
 
-from coaster.sqlalchemy import (TimestampMixin, BaseMixin, BaseNameMixin, BaseScopedNameMixin,
-    BaseScopedIdNameMixin, BaseIdNameMixin, MarkdownColumn, JsonDict, CoordinatesMixin)
+from coaster.sqlalchemy import (TimestampMixin, BaseMixin, BaseNameMixin,
+    BaseScopedNameMixin, BaseScopedIdNameMixin, BaseIdNameMixin, MarkdownColumn,
+    JsonDict, CoordinatesMixin, make_timestamp_columns)
```

Comment:
Missing `make_timestamp_columns` from import was making the tests fail as it was still being imported from here inside `label.py` https://github.com/hasgeek/funnel/blob/d2d1763faf83b46e89acdaddd00b16162245d239/funnel/models/label.py#L3